tensorflow/lite/micro/micro_interpreter.cc

```diff
@@ -31,8 +32,36 @@ limitations under the License.
 #include "tensorflow/lite/schema/schema_generated.h"
 #include "tensorflow/lite/schema/schema_utils.h"
 
+//#include "stdio.h"
```

Comment:
This seems like something left from debugging?

Reply:
reverted this change